packages/coinstac-graphql/src/data/resolvers.js

```diff
@@ -123,15 +123,20 @@ const resolvers = {
           rethink.table('consortia').insert(
             args.consortium,
             { 
-              conflict: "replace",
+              conflict: "update",
               returnChanges: true,
             }
           )
           .run(connection)
+          .then((result) => {
+            //r.db('coinstac').table('users').filter({id: 'test'}).update({'permissions': {'consortia': {'some-new-id': {'write': true}}}})
+            let crazything = {};
+            crazything[result.changes[0].new_val.id] = {'write': true};
+            rethink.table('users').filter({id: 'test'}).update({'permissions': {'consortia': crazything}})
```

Comment:
Doesn't seem like we should be hardcoding the `id` here... You should be able to get the user's id from the auth parameter. Just have the change the function a bit:

```
saveConsortium: ({ auth: { credentials } }, args) => {
  const { permissions } = credentials;
  if (!permissions.consortia.write
          && args.consortium.id
          && !permissions.consortia[args.consortium.id].write) {
            return Boom.forbidden('Action not permitted');
      }
  ...
  rethink.table('users').filter({ id: credentials.id }).update({'permissions': { consortia }})
```